src/socket.c

```diff
@@ -415,8 +425,8 @@ void pthreads_socket_connect(zval *object, zend_string *address, zend_long port,
 			struct sockaddr_un s_un = {0};
 
 			if (ZSTR_LEN(address) >= sizeof(s_un.sun_path)) {
-				/* throw */
-				return;
+				PTHREADS_THROW_MSG(0, "Path too long");
+				RETURN_FALSE;
```

Comment:
This `RETURN_FALSE` (and the other two above) could simply be `return;`, since the exception thrown makes the return value inaccessible.

Reply:
Yes, you're right.